MAPL_Shared/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+esma_set_this ()
+
+if(CMAKE_Fortran_COMPILER_ID STREQUAL "Intel")
+  if(CMAKE_Fortran_COMPILER_VERSION VERSION_LESS 20)
+    if(CMAKE_Fortran_COMPILER_VERSION VERSION_GREATER 17)
+      add_definitions(-D__ifort_18)
+    endif()
+  endif()
+endif()
+
```

Comment:
I don't think this logic is needed here.   That is something used with various gFTL containers.

Reply:
removed